packages/gatsby/cache-dir/slice.js

```diff
@@ -81,12 +85,7 @@ class SlicePropsError extends Error {
       stackLines[0] = stackLines[0].trim()
       stack = `\n` + stackLines.join(`\n`)
 
-      // look for any hints for the component name in the stack trace
-      const componentRe = /^at\s+([a-zA-Z0-9]+)/
-      const componentMatch = stackLines[0].match(componentRe)
-      const componentHint = componentMatch ? `in ${componentMatch[1]} ` : ``
-
-      message = `Slice "${sliceName}" was passed props ${componentHint}that are not serializable (${errors}).`
+      message = `Slice "${sliceName}" was passed props that are not serializable (${errors}).`
     } else {
       // we can't really grab any extra info outside of the browser, so just print what we can
       message = `${name}: Slice "${sliceName}" was passed props that are not serializable (${errors}). Use \`gatsby develop\` to see more information.`
```

Comment:
With the prod build now showing codeframe, maybe this should be applied:

```suggestion
      message = `${name}: Slice "${sliceName}" was passed props that are not serializable (${errors}).`
```

Reply:
I'll go ahead and apply this as develop doesn't really show more information